chapters/functions.tex

```diff
@@ -1502,6 +1502,8 @@ \subsection{Partial Derivatives of Functions}\label{partial-derivatives-of-funct
   h := Gibbs(p, T) - T * Gibbs_T(p, T);
 end specificEnthalpy;
 \end{lstlisting}
+Thus \lstinline!der(Gibbs, T)! corresponds to $\frac{\partial \text{\lstinline!Gibbs!}}{\partial \text{\lstinline!T!}}=\frac{\partial}{\partial \text{\lstinline!T!}}\text{\lstinline!Gibbs!}$.
+For multiple arguments \lstinline!der(Gibbs, T, T, p)! corresponds to $\frac{\partial}{\partial \text{\lstinline!p!}}\frac{\partial}{\partial \text{\lstinline!T!}}\frac{\partial}{\partial \text{\lstinline!T!}}\text{\lstinline!Gibbs!}$.
```

Comment:
With the more extensive normative test, I think this sentence can be dropped from the example:

```suggestion
```